internal/release/promote/commits_metadata.go

```diff
@@ -0,0 +1,46 @@
+package promote
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+type CommitMetadata struct {
+	Sha     string
+	Author  string
+	Message string
+}
+
+func GetCommitsMetadata(dir, from, to string) ([]*CommitMetadata, error) {
```

Comment:
This is not a request for change, but these types of string parsing functions make me think that as soon as we get some bandwidth for maintenance week, we should introduce `go-git`

Reply:
Yeah, totally! 👍